08-CMSIS/core/src/itoa.c

```diff
@@ -0,0 +1,20 @@
+void itoa(int i, char *a)
```

Comment:
Since `itoa` is only used by one C source, it is not worth to implement as public function.